### DISCUSSION TITLE
uncurryThis uses bind twice

### DISCUSSION THREAD

**Discussion by saurabhraushan** [nodejs/help#4811] [Q&A] [answered]
i don't understand this line plz help

**Comment by JimmyDaddy** [comment]
bro, could you please paste the sample code here? This will help us understand your question better.

**Comment by aduh95** [comment, marked as answer]
I don't know if it helps, but it used to be even more convoluted: https://github.com/nodejs/node/pull/36866

I wouldn't worry too much if you don't understand it, no one should need to make their own `uncurryThis`, if you're happy to take my word for it, just know it's equivalent to `const uncurryThis = (fn) => (this, ...args) => Reflect.apply(fn, this, args)`.
